lib/msal-node/src/index.ts

```diff
@@ -22,4 +22,6 @@ export {
     AuthError,
     AuthErrorMessage,
     INetworkModule,
+    // Logger
+    LogLevel,
```

Comment:
Should `Logger` also be exported?

Reply:
@jasonnutter The way that node is set up, the client only needs to configure logger options, with the callback, pii, and logLevel, without having to create an instance of Logger on the client side (unlike other libraries). The Logger instance is created by node in the ClientApplication.ts file. Not sure where the client would use Logger if it was exported, so I didn't include it.

Reply:
Ah okay, makes sense.